bindings/python/test.py

```diff
@@ -204,6 +204,11 @@ def test_read_resume_data(self):
 
         ses = lt.session({'alert_mask': lt.alert.category_t.all_categories})
         h = ses.add_torrent(tp)
+        print('add_torrent_params:')
```

Comment:
Maybe it would be best to try accessing all of the properties to ensure none of them break in the future?

for attr in dir(tp):
  print('%s: %s' % (attr, getattr(tp, attr)))